src/markdown-pages/contribute-to-quickstarts/instrument-library/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Instrument your Library'
+template: 'OverviewTemplate'
+description: ''
+---
+<Intro>
+If your looking to jump into building your own quickstart, the first step is instrumenting your own library to send data to New Relic. New Relic I/O gives end users the ability to slice and dice observability data around their application, server, etc. Setting up your library to get data into New Relic is critical to this process, so that the end users have a streamlined click-thorugh experience to see your data on our platform. You will need to instrument your library with at least one data type to be able to observe it via a dashboard or alert.
+</Intro>
+
+## Observability Data Options
+
+ New Relic has [four types of observability data](https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types) which we call MELT data; *Metrics, Events, Logs, and Traces*. Each one handles a specific use case, so you'll have to decide which one fits your needs best.
```

Comment:
> Each one handles a specific use case, so you'll have to decide which one fits your needs best.

I'm not sure I follow. How does "metrics" handle a specific use case? Maybe you mean something more like:

> Each one serves a different purpose in providing visibility into your library's behaviors and performance.

Or something like that?

---

> New Relic has [four types of observability data]

I would be careful about jargon like "observability" here. Many people don't know what this means, so a brief definition would be valuable. Later on in this PR, you use others like:

- log drains
- agent
- NRQL
- etc.

These terms may not be clear to your target audience. Sometimes it's best to remove them in favor of something simpler. If you can't, than an explanation is valuable in providing context.

Reply:
I believe I was able to resolve these issues with the last commit, but I'll leave this open.